src/index.html

```diff
@@ -11,7 +11,186 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      autocomplete="off"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <div class="input">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+          >
+        </div>
+
+        <div class="input">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            id="name"
+            name="name"
+            required
+          >
+        </div>
+
+        <div class="input">
+          <label for="old">How old are You?</label>
+          <input
+            type="number"
+            id="old"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </div>
+
+        <div class="input">
+          <label for="birth">Full date of birth:</label>
+          <input
+            type="date"
+            id="birth"
+            name="birth"
+          >
+        </div>
+
+        <div>
+          <label for="agreement">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            id="agreement"
+            name="agreement"
+          >
+        </div>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <div class="input">
+          <label for="email" >E-mail:</label>
```

Comment:
Check all your work and fix all cases if present
```suggestion
          <label for="email">E-mail:</label>
```